frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersionsTable.tsx

```diff
@@ -23,8 +23,10 @@ const ModelVersionsTable: React.FC<ModelVersionsTableProps> = ({
   isArchiveModel,
   refresh,
 }) => {
-  const { registeredModelId } = useParams();
-  const inferenceServices = useMakeFetchObject(useInferenceServices(undefined, registeredModelId));
+  const { mrName, registeredModelId } = useParams();
+  const inferenceServices = useMakeFetchObject(
+    useInferenceServices(undefined, registeredModelId, mrName),
```

Comment:
Same here, I think you need an `undefined` for the version ID so you push `mrName` into the 4th argument position.